src/App.js

```diff
@@ -1,14 +1,33 @@
 import "./App.css";
-import React from "react";
-import Header from "./components/Header";
+import React, { useEffect, useState } from "react";
+import Title from "./components/Header";
 import Card from "./components/Card";
 import NavBar from "./components/Navbar";
 
 function App() {
+  const [characters, setCharacters] = useState([]);
+  const url = "https://rickandmortyapi.com/api/character";
+
+  const loadCharacters = () => {
+    fetch(url)
+      .then((response) => response.json())
+      .then((data) => setCharacters(data.results));
+  };
+
+  useEffect(() => {
+    loadCharacters();
+  }, []);
+
   return (
     <div className="App">
-      <Header />
-      <Card />
+      <Title />
+      {characters.map((character) => (
+        <Card
+          key={character.id}
+          image={character.image}
+          name={character.name}
+        />
+      ))}
```

Comment:
You need to add the `ul` element in your App and not in your `Card`.

```
<ul role="list">
      {characters.map((character) => (
        <Card
          key={character.id}
          image={character.image}
          name={character.name}
        />
      ))}
</ul>
```
If you add the `ul` element in your `Card` component then every card has an own `ul` but we want to have all cards in one `ul`. I hope this makes it clear now, as I think my explanation before was misunderstanding. 🙏

Reply:
thank u so much. I would say the confusion was not made by ur explanation rather than by my leck of experienced. :)